### PR TITLE
adding genomad version 1.8.0

### PR DESCRIPTION
**Pull request by erinyoung**
There's a new version of geNomad!

I coped the files from 1.7.4 and made the following changes:
1. updated the software version arg
2. changed the download path for aragorn (because the old one doesn't exist anymore)
3. added some commented lines in test to use in case end-to-end doesn't work (the test stage currently hangs in gitpod, but I'm hopeful it'll be fine in github actions)
4. updated the database version in the readme

```
$ diff genomad/1.7.4/Dockerfile genomad/1.7.5/Dockerfile 
2c2
< ARG GENOMAD_VER="1.7.4"
---
> ARG GENOMAD_VER="1.7.5"
18c18
< RUN wget -q https://www.ansikte.se/ARAGORN/Downloads/aragorn${ARAGORN_VER}.c && \
---
> RUN wget -q https://www.trna.se/ARAGORN/Downloads/aragorn${ARAGORN_VER}.c && \
82a83,89
> #    genomad annotate GCA_025259185.1_ASM2525918v1_genomic.fna genomad_output /genomad_db && \
> #    genomad find-proviruses GCA_025259185.1_ASM2525918v1_genomic.fna genomad_output /genomad_db && \
> #    genomad marker-classification GCA_025259185.1_ASM2525918v1_genomic.fna genomad_output /genomad_db && \
> #    genomad nn-classification GCA_025259185.1_ASM2525918v1_genomic.fna /genomad_db && \
> #    genomad aggregated-classification GCA_025259185.1_ASM2525918v1_genomic.fna /genomad_db && \
> #    genomad score-calibration GCA_025259185.1_ASM2525918v1_genomic.fna /genomad_db && \
> #    genomad summary GCA_025259185.1_ASM2525918v1_genomic.fna /genomad_db
```

Pull Request (PR) checklist:
- [X] Include a description of what is in this pull request in this message.
- [X] The dockerfile successfully builds to a test target for the user creating the PR. (i.e. `docker build --tag samtools:1.15test --target test docker-builds/samtools/1.15` )
- [X] Directory structure as name of the tool in lower case with special characters removed with a subdirectory of the version number (i.e. `spades/3.12.0/Dockerfile`)
   - [X] (optional) All test files are located in same directory as the Dockerfile (i.e. `shigatyper/2.0.1/test.sh`)
- [X] Create a simple container-specific [README.md](https://github.com/StaPH-B/docker-builds/blob/master/.github/workflow-templates/readme-template.md) in the same directory as the Dockerfile (i.e. `spades/3.12.0/README.md`)
   - [X] If this README is longer than 30 lines, there is an explanation as to why more detail was needed
- [X] Dockerfile includes the recommended [LABELS](https://github.com/StaPH-B/docker-builds/blob/master/dockerfile-template/Dockerfile#L8-L18) 
- [X] Main [README.md](https://github.com/StaPH-B/docker-builds/blob/master/README.md) has been updated to include the tool and/or version of the dockerfile(s) in this PR
- [X] [Program_Licenses.md](https://github.com/StaPH-B/docker-builds/blob/master/Program_Licenses.md) contains the tool(s) used in this PR and has been updated for any missing

### CODE REVIEW

**Comment by erinyoung** [comment]
This is finally ready for review! I needed to tag keras and tensorflow to a specific version.

**Comment by erinyoung** [comment]
This means there are more differences in the dockerfiles:

```
$ diff genomad/1.7.4/Dockerfile genomad/1.7.5/Dockerfile 
2c2
< ARG GENOMAD_VER="1.7.4"
---
> ARG GENOMAD_VER="1.7.5"
4a5,6
> ARG KERAS_VER="2.15.0"
> ARG TENSORFLOW_VER="2.15.0"
18c20
< RUN wget -q https://www.ansikte.se/ARAGORN/Downloads/aragorn${ARAGORN_VER}.c && \
---
> RUN wget -q https://www.trna.se/ARAGORN/Downloads/aragorn${ARAGORN_VER}.c && \
25a28,29
> ARG KERAS_VER
> ARG TENSORFLOW_VER
55c59
<     python3 -m pip install v${GENOMAD_VER}.tar.gz && \
---
>     python3 -m pip install v${GENOMAD_VER}.tar.gz keras==${KERAS_VER} tensorflow==${TENSORFLOW_VER} && \
59c63,64
< ENV PATH="/mmseqs/bin:$PATH" LC_ALL=C
---
> ENV PATH="/mmseqs/bin:$PATH" LC_ALL=C 
> #CUDA_VISIBLE_DEVICES=-1
81c86
< RUN wget -q https://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/025/259/185/GCA_025259185.1_ASM2525918v1/GCA_025259185.1_ASM2525918v1_genomic.fna.gz && \
---
> RUN pip list && wget -q https://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/025/259/185/GCA_025259185.1_ASM2525918v1/GCA_025259185.1_ASM2525918v1_genomic.fna.gz && \
83c88,95
<     genomad end-to-end GCA_025259185.1_ASM2525918v1_genomic.fna test /genomad_db
\ No newline at end of file
---
> #    genomad annotate GCA_025259185.1_ASM2525918v1_genomic.fna genomad_output /genomad_db && \
> #    genomad find-proviruses GCA_025259185.1_ASM2525918v1_genomic.fna genomad_output /genomad_db && \
> #    genomad marker-classification GCA_025259185.1_ASM2525918v1_genomic.fna genomad_output /genomad_db && \
> #    genomad nn-classification GCA_025259185.1_ASM2525918v1_genomic.fna /genomad_db && \
> #    genomad aggregated-classification GCA_025259185.1_ASM2525918v1_genomic.fna /genomad_db && \
> #    genomad score-calibration GCA_025259185.1_ASM2525918v1_genomic.fna /genomad_db && \
> #    genomad summary GCA_025259185.1_ASM2525918v1_genomic.fna /genomad_db
>     genomad end-to-end GCA_025259185.1_ASM2525918v1_genomic.fna test /genomad_db --splits 16
\ No newline at end of file
```

**Comment by erinyoung** [comment]
Version 1.7.6 came out, so I've updated all the files in this PR. 

There's nothing really new about this version. It's more for package managers to ensure that the version of keras is under 3.0. https://github.com/apcamargo/genomad/releases/tag/v1.7.6

**Review by Kincekara — APPROVED** [review]

**Comment by Kincekara** [comment]
Thank you for putting all these together! Apologies for the long wait... 
Deploy link: https://github.com/StaPH-B/docker-builds/actions/runs/9400988759
